NZSLDict/Classes/SearchViewController.swift

```diff
@@ -350,7 +373,7 @@ class SearchViewController: UIViewController, UISearchBarDelegate, UITableViewDa
         if sender.state == .Ended {
             self.selectEntry(wordOfTheDay)
             searchBar.resignFirstResponder()
-            self.delegate.didSelectEntry(wordOfTheDay)
+            self.delegate?.didSelectEntry(wordOfTheDay)
```

Comment:
what's the `?` doing for us? Is it just a method name like in ruby -- or something magic?